src/main/java/pl/put/poznan/sorting/logic/Timer.java

```diff
@@ -1,36 +1,15 @@
 package pl.put.poznan.sorting.logic;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import pl.put.poznan.sorting.app.SortingMadness;
-
-/**

```

Comment:
To tu musi pozostać. Bez tego javadoc nie wygeneruje dokumentacji dla tej klasy.